addons/sale/static/src/js/variant_mixin.js

```diff
@@ -581,6 +581,14 @@ var VariantMixin = {
                 combination.carousel,
                 isCombinationPossible
             );
+
+            const $variantsContainer = ev.target.closest('.js_add_cart_variants');
+            if(ev.target.tagName === 'INPUT' && $variantsContainer){
+                $variantsContainer.scrollIntoView({
+                    behavior: 'auto',
+                    block: 'center',
+                });
+            }
```

Comment:
I think this should be in the `variant_mixin` of `website_sale`. 

If I understood correctly the problem, this has nothing to do with `sale` as the error is only present in eCommerce also this file was dropped in `saas-16.3` so you'll have conflicts if you keep your code there.

Reply:
@chevalierv you are totally right ! thank you for the feedback :pray: